Makefile

```diff
@@ -0,0 +1,36 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+SHELL := /bin/bash -o pipefail
+
+VERSION ?= latest
+RELEASE_SRC = apache-apisix-java-plugin-runner-${VERSION}-src
+
+.PHONY: release-src
+release-src: compress-tar
+	gpg --batch --yes --armor --detach-sig ./dist/$(RELEASE_SRC).tgz
```

Comment:
I try `VERSION=0.1 make release-src`,
get:

```
gpg --batch --yes --armor --detach-sig ./dist/apache-apisix-java-plugin-runner-0.1-src.tgz
gpg: can't open './dist/apache-apisix-java-plugin-runner-0.1-src.tgz': No such file or directory
```

Reply:
can try `make release-src VERSION=0.1.0-SNAPSHOT`

Reply:
Need to doc it in the Makefile

Reply:
update `VERSION ?= latest`  to `VERSION ?= 0.1.0-SNAPSHOT`